### ISSUE TITLE
Update What's new in .NET 9 for RC 2

### ISSUE THREAD

**Issue by gewarren** [dotnet/docs#41288]
### Describe the issue or suggestion

Update What's new in .NET 9.

**Comment by takasurazeem** [comment]
Is it released yet? Can I download it for macOS?

**Comment by gewarren** [comment]
> Is it released yet? Can I download it for macOS?

Yes! You can download RC 1 here: https://dotnet.microsoft.com/en-us/download/dotnet/9.0
